src/main/java/uk/gov/hmcts/divorce/solicitor/service/CcdAccessService.java

```diff
@@ -171,7 +171,7 @@ public boolean isApplicant2(String userToken, Long caseId) {
 
     public void removeUsersWithRole(Long caseId, List<String> roles) {
         final var userDetails = idamService.retrieveSystemUpdateUserDetails().getUserDetails();
-        log.info("user email: {}, id: {}", userDetails.getEmail(), userDetails.getId());
+        log.info("user id: {}", userDetails.getSub(), userDetails.getUid());
```

Comment:
you missed one

Reply:
I intended to delete it because we shouldn't log emails. Will clean it up